src/Migrations.php

```diff
@@ -66,6 +75,28 @@ public function __construct(array $default = [])
     }
 
     /**
+     * Sets the command
+     *
+     * @param string $command Command name to store.
+     * @return self
```

Comment:
So far we have been using $this.
But I would favor if we migrated to `self` for CakePHP in the near future.